static/app/views/onboarding/setupDocs.tsx

```diff
@@ -1,4 +1,4 @@
-import {Fragment, useCallback, useEffect, useMemo, useState} from 'react';
+import React, {Fragment, useCallback, useEffect, useMemo, useState} from 'react';
```

Comment:
nit: you don't need to import `React` for types

Reply:
oh yes, missing this one...thanks

Reply:
do we have a way to remove them automatically from the imports? 🤔

Reply:
Can we lint for this style nit?

I've seen the comment "you don't need to import React for types" and the similar "you don't need to import types from react, use React.ReactNode directly" too often that, if it's really a real problem we should deal with it.

But on the other hand, it's so common that people don't seem to mind doing the import. We have to import Organization so often, treating react as a special case might be more overhead.